### PR TITLE
Updated VC Transition Animation

### PR DESCRIPTION
**Pull request by smiller194**
Updated transition animation handling to meet requirements set by PM-12185

https://wbdigital.atlassian.net/browse/PM-12185

This is needed to merge the accompanying ww ticket

### CODE REVIEW

File: Source/FolioReaderCenter.swift [hunk 1]
```diff
@@ -8,7 +8,9 @@
 
 import UIKit
 //import ZFDragableModalTransition
-
+public extension Notification.Name {
+    static let pageDidLoadNotif = Notification.Name("pageDidAppearNotif")
```
Comment (yuweipei) [inline Source/FolioReaderCenter.swift]: `pageDidLoadNotification` sounds about right to me, maybe we don't need the abbreviation.

File: Source/FolioReaderKit.swift [hunk 2]
```diff
@@ -169,8 +173,12 @@ extension FolioReader {
         Bool = true) {
         let readerContainer = FolioReaderContainer(withConfig: config, folioReader: self, epubPath: epubPath, unzipPath: unzipPath, decryptionKey: decryptionKey, removeEpub: shouldRemoveEpub)
         self.readerContainer = readerContainer
+        let folioReaderTransitionDelegate = FolioReaderTransitionDelegate()
         readerContainer.modalPresentationStyle = .fullScreen
-        parentViewController.present(readerContainer, animated: animated, completion: nil)
+        readerContainer.transitioningDelegate = folioReaderTransitionDelegate
```
Comment (yuweipei) [inline Source/FolioReaderKit.swift]: can we combine `176` and `178` into one line? 
`readerContainer.transitioningDelegate = FolioReaderTransitionDelegate()`

File: Source/FolioReaderKit.swift [hunk 3]
```diff
@@ -23,6 +23,10 @@ internal let kCurrentTOCMenu = "com.folioreader.kCurrentTOCMenu"
 internal let kHighlightRange = 30
 internal let kReuseCellIdentifier = "com.folioreader.Cell.ReuseIdentifier"
 
+public extension Notification.Name {
+    static let folioReaderPresentationComplete = Notification.Name("folioReaderPresentationComplete")
```
Comment (yuweipei) [inline Source/FolioReaderKit.swift]: i think `folioReaderPresentationCompleted` is better than `folioReaderPresentationComplete`

File: Source/FolioReaderCenter.swift (hunk 1 above)
Comment (smiller194) [reply Source/FolioReaderCenter.swift]: name changed to pageDidLoadNotification

File: Source/FolioReaderKit.swift (hunk 2 above)
Comment (smiller194) [reply Source/FolioReaderKit.swift]: lines combined

File: Source/FolioReaderKit.swift (hunk 3 above)
Comment (smiller194) [reply Source/FolioReaderKit.swift]: name changed to folioReaderPresentationCompleted

**Pull request merged**